src/models/Factor.js

```diff
@@ -207,7 +207,7 @@ function (Okta, Q, factorUtil, BaseLoginModel) {
         }
 
         // MFA_CHALLENGE
-        else if (this.get('canUseResend') && transaction.resend) {
+        else if (this.get('canUseResend') && !this.get('answer') && transaction.resend) {
```

Comment:
but you shouldn't use resend to validate the answer. We should just be doing another post to the resend link.
Check the Form see if there is something that is wrong there.